labs-ci-cd.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
as  you are only calling the `oc-apply` role, I'd suggest using the playbook in `casl-ansible` instead to avoid tracking the role specifically - i.e.: https://github.com/redhat-cop/casl-ansible/blob/master/playbooks/openshift-cluster-seed.yml